modules/rhpam-apb/roles/deploy-amq/tasks/main.yml

```diff
@@ -26,6 +26,21 @@
     - name: "[AMQ] configure route"
       include_tasks: route.yml
 
+    - name: "[AMQ] generate secret if not present"
+      include_role:
+        name: generate-secret
+      vars:
+        amq_secret: true
```

Comment:
Remove this flag

Reply:
why exactly you are suggesting to remove it? I added this to execute the amq certs configuration only for amq.
Suggestions about another way to do this?